### PR TITLE
Note to FlexJobs

### PR DESCRIPTION
**Pull request by sharktrexer**
See #867

This jobs site also has a lot of negative reviews. Should honestly be removed but this note should help.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -149,7 +149,7 @@ A curated list of awesome [remote working](https://en.wikipedia.org/wiki/Telecom
   1. [Dribbble Jobs](https://dribbble.com/jobs?location=Anywhere)
   1. [Drupal Jobs](https://jobs.drupal.org/home/type/telecommute-remote-3588)
   1. [Find Bacon](https://findbacon.com) - Design and Dev jobs
-  1. [Flexjobs](https://www.flexjobs.com/) – Telecommuting Jobs & Professional Part-Time Jobs.
+  1. [Flexjobs](https://www.flexjobs.com/) – Telecommuting Jobs & Professional Part-Time Jobs. Requires paid membership to see all results.
```
Comment (papali1305) [inline README.md:152]: why file is required

Comment (sharktrexer) [reply README.md:152]: Do you mean why this change should exist? I think when you have a list of job boards there should be some transparency that a board will require $$$ to properly use.

**Review by Technetium1 — APPROVED** [review]
This is a necessary change.

File: README.md (hunk 1 above)
Comment (GitHub7667) [reply README.md:152]: Nice👍👏

**Comment by RichardLitt** [comment]
Just remove it and then @RichardLitt me to merge.

**Review by RichardLitt — APPROVED** [review]

**Pull request merged**